schemas/2015-03-01-preview/Microsoft.AppService.json

```diff
@@ -172,31 +160,6 @@
         "properties"
       ]
     },
-    "resourceReference": {
-      "type": "object",
-      "description": "Reference to a CSM resource",
-      "properties": {
-        "id": {
-          "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression",
-          "description": "Full CSM resource ID"
-        },
-        "resourceName": {
-          "oneOf": [
-            {
-              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-            },
-            {
-              "type": "string"
-            }
-          ],
-          "description": "CSM resource name"
-        },
-        "resourceType": {
-          "type": "string",
-          "description": "Type of a CSM resource"
-        }
-      }
-    },
     "appServiceApiVersion": {
```

Comment:
"apiServiceApiVersion" is not a resource, so it should be under "definitions", right?